### PR TITLE
2. New data management/actions and provider

### PR DESCRIPTION
**Pull request by codingki**
Part of #452

## Description

actions for every contract abi 

## Changes

- [x] [feat: compositeDataProvider react query provider](https://github.com/strangelove-ventures/sommelier/commit/cc9f4223542684c754d7880e2bfbe822fbb5f05b)
- [x] [chore: cellar action for AAVE_V2_STABLE_CELLAR contract](https://github.com/strangelove-ventures/sommelier/commit/e1215dc210ded9e3f8f5e72f585937ea6f4b8078)
- [x] [chore: staker action for AAVE_STAKER contract](https://github.com/strangelove-ventures/sommelier/commit/4cf25aa661a3382a79c138737428f7f63122f611)

### CODE REVIEW

File: src/composite-data/actions/staker/types.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import BigNumber from "bignumber.js"
+import { BigNumber as BigNumberE } from "ethers"
+
+export interface UserStake {
+  amount: BigNumberE
```
Comment (sandbochs) [inline src/composite-data/actions/staker/types.ts:5]: I wonder if this is a good time to switch the BigNumberE usage to BigNumber

Comment (sandbochs) [reply src/composite-data/actions/staker/types.ts:5]: Probably a way bigger refactor tho

Comment (codingki) [reply src/composite-data/actions/staker/types.ts:5]: Yeah, I'll do it after 5 of these PR merged

File: src/composite-data/actions/cellar/AAVE_V2_STABLE_CELLAR/fetchCellarShareBalance.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  try {
+    if (!aaveClrBalance) {
+      throw new Error("aaveClrBalance is undefined")
+    }
+    if (!totalBondedAmount) {
+      throw new Error("totalBondedAmount is undefined")
+    }
+
+    const cellarSharebalance = await contract.convertToAssets(
+      new BigNumber(aaveClrBalance).plus(totalBondedAmount).toFixed()
```
Comment (xanderjl) [inline src/composite-data/actions/cellar/AAVE_V2_STABLE_CELLAR/fetchCellarShareBalance.ts:22]: Only suggestion I'd make here is to define this BigNumber as its own variable above, just helps with readability a little bit :)

File: src/composite-data/actions/staker/types.ts (hunk 1 above)
Comment (xanderjl) [reply src/composite-data/actions/staker/types.ts:5]: This makes sense to me. Would be nice to migrate away from `bignumber.js` and use `ether`'s package, but I'm sure there's a use case I've not considered here. @sandbochs Any insight into that? I feel like we discussed this with Lucky a long while ago, but I can't recall his answer

**Review by xanderjl — APPROVED** [review]
My one suggestion is definitely not a blocker, just food for thought. Great work @codingki 💪

**Pull request merged**